src/main/java/com/faforever/client/preferences/ui/SettingsController.java

```diff
@@ -255,9 +286,29 @@ private void configureThemeSelection(Preferences preferences) {
     );
   }
 
-  private void configureLanguageSelection() {
-    languageComboBox.setItems(FXCollections.singletonObservableList("English"));
-    languageComboBox.getSelectionModel().select(0);
+  private void configureLanguageSelection(Preferences preferences) {
+    languageComboBox.setItems(FXCollections.observableArrayList(LanguageInfo.getAllDisplayNames()));
+    String language = preferences.getLanguage().getLanguage();
```

Comment:
`getLanguage().getLanguage()` bugs me somehow :P can we rename one of it? (probably the first)